custom/enikshay/ucr/tests/test_episode_datasource.py

```diff
@@ -1,207 +0,0 @@
-import os
-import mock
-from datetime import datetime
-
-from django.test import TestCase, override_settings
-from corehq.util.test_utils import TestFileMixin
-from corehq.form_processor.tests.utils import FormProcessorTestUtils
-
-from corehq.apps.userreports.const import UCR_SQL_BACKEND, UCR_ES_BACKEND
-from corehq.apps.userreports.util import get_indicator_adapter
-from corehq.apps.userreports.models import StaticDataSourceConfiguration
-from corehq.apps.userreports.tasks import rebuild_indicators
-from casexml.apps.case.const import CASE_INDEX_EXTENSION
-from casexml.apps.case.mock import CaseFactory, CaseStructure, CaseIndex
-
-
-class BaseEnikshayDatasourceTest(TestCase, TestFileMixin):
-    file_path = ('data_sources', )
-    root = os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
-    _call_center_domain_mock = mock.patch(
-        'corehq.apps.callcenter.data_source.call_center_data_source_configuration_provider'
-    )
-    datasource_filename = ''
-
-    @classmethod
-    def setUpClass(cls):
-        super(BaseEnikshayDatasourceTest, cls).setUpClass()
-        cls._call_center_domain_mock.start()
-        cls.static_datasource = StaticDataSourceConfiguration.wrap(
-            cls.get_json(cls.datasource_filename)
-        )
-        cls.domain = cls.static_datasource.domains[0]
-        cls.datasource = StaticDataSourceConfiguration._get_datasource_config(
-            cls.static_datasource,
-            cls.domain,
-        )
-        cls.factory = CaseFactory(domain=cls.domain)
-
-    @classmethod
-    def tearDownClass(cls):
-        super(BaseEnikshayDatasourceTest, cls).tearDownClass()
-        cls._call_center_domain_mock.stop()
-
-    def tearDown(self):
-        FormProcessorTestUtils.delete_all_cases()
-
-    def _get_query_object(self):
-        adapter = get_indicator_adapter(self.datasource)
-        return adapter.get_query_object()
-
-
-@override_settings(OVERRIDE_UCR_BACKEND=UCR_SQL_BACKEND)
-class TestEpisodeDatasource(BaseEnikshayDatasourceTest):
```

Comment:
It'd probably be better to update this test to use one of the new versions instead of removing tests